buzzmobile/constants.yaml

```diff
@@ -44,7 +44,7 @@ wheel_separation: 1.3716 # distance between left and right wheels in meters
 max_steering_angle: 0.262 # radians (=15 degrees)
 
 # Max speed of buzzmobile in m/s
-max_speed: 0.2
+max_speed: 2 #TODO this is a murder machine
```

Comment:
IIRC we put 0.2 down because the arduino software we got from RJ had a few bugs where in it would flush the speed update messages all at once, causing toyota mode to happen. We fixed it for when it was going forward, but didn't have time to do anything when it was turning.

You can experiment with this speed when you have access to the car, but I'd start with slow speed at first so you don't die while trying to figure out what the ideal speed is.